framework/widgets/DetailView.php

```diff
@@ -205,7 +205,7 @@ protected function normalizeAttributes()
 
         foreach ($this->attributes as $i => $attribute) {
             if (is_string($attribute)) {
-                if (!preg_match('/^([\w\.]+)(:(\w*))?(:(.*))?$/', $attribute, $matches)) {
+                if (!preg_match('/^([^:]+)(:(\w*))?(:(.*))?$/', $attribute, $matches)) {
```

Comment:
The updated regular expression will allow attribute names, starting from the digit, e.g. `5nizza'. Is not allowed it PHP.